runtime/lua/vim/pack.lua

```diff
@@ -162,8 +162,12 @@ local function git_cmd(cmd, cwd)
   return (stdout:gsub('\n+$', ''))
 end
 
+local git_version = vim.version.parse('1')
+
 local function git_ensure_exec()
-  if vim.fn.executable('git') == 0 then
+  local sys_res = vim.system({ 'git', 'version' }):wait()
+  git_version = vim.version.parse(sys_res.stdout)
+  if sys_res.stderr ~= '' then
```

Comment:
These lines don't seem to work as intended? If there is no `git` executable, already the first line (line 168) fails. It seems that line 170 is not reached?

The error message that appears is basically the same as when running `:lua vim.system({'does_not_exists'}):wait()`.